package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "devDependencies": {
+      "babel-eslint": "^10.1.0",
+      "eslint": "^7.32.0",
+      "eslint-config-airbnb-base": "^14.2.1",
+      "eslint-plugin-import": "^2.24.2",
+      "hint": "^6.1.4",
+      "stylelint": "^13.13.1",
+      "stylelint-config-standard": "^21.0.0",
+      "stylelint-csstree-validator": "^1.9.0",
+      "stylelint-scss": "^3.20.1"
+    }
+  }
```

Comment:
- [ ] There is no script defined here to start the app. Let's set up the different scripts to run the app and tests.